performance/tests/VirtualListClientSize.test.js

```diff
@@ -0,0 +1,86 @@
+
+const puppeteer = require('puppeteer');
+const {Mount} = require('../TraceModel');
+const {getFileName} = require('../utils');
+const TestResults = require('../TestResults');
+
+describe('VirtualList clientSize prop', () => {
+	let browser;
+	let page;
+
+	beforeEach(async () => {
+		browser = await puppeteer.launch({headless: true});
+		page = await browser.newPage();
+		await page.setViewport({
+			width: 1920,
+			height: 1080
+		});
+
+		const client = await page.target().createCDPSession();
+		await client.send('Emulation.setCPUThrottlingRate', {rate: 6});
+	});
+
+	afterEach(async () => {
+		await browser.close();
+	});
+
+	describe('VirtualList js type', () => {
+		it('mount time without clientSize', async () => {
+			const filename = getFileName('VirtualList_withOutClientSize');
+
+			await page.tracing.start({path: filename, screenshots: false});
+			await page.goto('http://localhost:8080/virtualListClientSize?type=JS');
```

Comment:
I have one question for the route path. Is there any reason to start a lower character for the route path here? For example, not `virtualListClientSize` but `VirtualListClientSize`.

Reply:
No specific reason. It's more of a convention to use lowercase for path but we are using camel case ;)

Reply:
I got it.